### PR TITLE
Birmingham Class 2 - Mame_Mbaye - Git and Github

### PR DESCRIPTION
**Pull request by mame-baba**
# Your Details

- Your Name: Mame Mbaye
- Your City: Birmingham
- Your Slack Name: Mame MB

# Homework Details

- Module: Git and Github
- Week: 2

### CODE REVIEW

**Review by Know-Thyself — APPROVED** [review]
I believe you have achieved the main objectives of the assignment in terms of managing to clone, commit, push and pule using Git and GitHub.
However, you could add one more problem Git & GitHub Can solve.

**Review by SenaitEkubai — COMMENTED** [review]
Nice work Mame. I think you have forgotten to fill the details in the file. I did forget to fill my details too but i filled them following Aaron's comment.

File: homework.md [hunk 1]
```diff
@@ -4,41 +4,37 @@
...
 ## Question 6
 
 Using CSS, how would I select every `<h1>` tag on the page?
 
-```css
-#h1 {
-
-}
-```
+h1{}
```
Comment (jip1984) [inline homework.md:37]: h1 {
    color: #fff;
}

**Comment by mame-baba** [comment]
Hi Senait,
Thank you for your comment, much appreciated. You are right, I will correct
it now.
Please do not hesitate to make a comment if anything else is
wrong/forgotten.
Many thanks
Mame

**Comment by mame-baba** [comment]
Hi Jon,
Thank you for your comment.
Definitely missed that one, Thank you for highlighting and correcting it.
Many Thanks
Mame

File: homework.md (hunk 1 above)
Comment (mame-baba) [reply homework.md:37]: Perfect

**Pull request closed without merging**